### DISCUSSION TITLE
WebGIS 6 Feedback

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#179] [General]
Build 6.24.2602:
**Bilder/Icons**, die über folgende Methode aufgerufen werden, werden nicht angezeigt:
`.../wgapi/rest/imageresource/user-24~png?sub=&company=gisstmk&colorscheme=default&v=6.24.2602`
`.../wgapi/rest/imageresource/error-80~png?sub=&company=gisstmk&colorscheme=default&v=6.24.2602`
in den DevTools kommt folgende Fehlermeldung
_500 (Internal Server Error)_

Die Bilder sind aber im Ordner `\wwwroot\content\api\img` vorhanden.

**Zeichnen/Redlining**
Zeichnen von Symbolen: Verwendet wird immer das Symbol an der ersten Stelle. Wenn man dann die Symbolik ändern will, ist die "Seite" leer.
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/45ced4ff-e933-4bae-8568-4d1f6ff6befe)
Über devTools sieht man, dass für diesen Bereich `Display:none` eingestellt wird. Ändert man es auf `Display:visible`, dann erscheinen die Symbole und man kann die Symbolik ändern.

**Comment by gis-eni** [comment]
Kann ich beides nachvollziehen und ist dann im nächsten Build behoben, danke
